Cargo.toml

```diff
@@ -8,12 +8,13 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [features]
-default = [ "openssl", "threading" ]
+default = [ "pkgconfig", "openssl", "threading" ]
 static = [ "libevent-sys/static" ]
+pkgconfig = [ "libevent-sys/pkgconfig" ]
```

Comment:
If a package is optional you can include it the same as you'd include a feature - so this can just be `libevent-sys/pkg-config` and you can remove the named feature in the `libevent-sys` crate. Additionally you could just add the feature `libevent-sys/pkg-config` to your `default` config and get rid of all the `pkgconfig` aliases.